core/components/css-utilities/Border/Border.story.tsx

```diff
@@ -129,50 +218,323 @@ export const border = () => {
-        {'<span className="border border-0 bg-secondary-lightest p-8 d-inline-block mr-6">border-0</span>'}
-        <br />
-        {'<span className="border border-top-0 bg-secondary-lightest p-8 d-inline-block mr-6">top-0</span>'}
-        <br />
-        {'<span className="border border-right-0 bg-secondary-lightest p-8 d-inline-block mr-6">right-0</span>'}
-        <br />
-        {'<span className="border border-bottom-0 bg-secondary-lightest p-8 d-inline-block mr-6">bottom-0</span>'}
-        <br />
-        {'<span className="border border-left-0 bg-secondary-lightest p-8 d-inline-block">left-0</span>'}
+
+      {/* Border style examples */}
+      <div className="mt-8 mb-8">
+        <Heading size="s" className="mb-3">
+          Border Styles
+        </Heading>
+        <div className="mb-4">
+          <Text appearance="default" className="border border-solid bg-secondary-lightest p-8 d-inline-block mr-6">
+            border-solid
+          </Text>
+          <Text appearance="default" className="border border-dashed bg-secondary-lightest p-8 d-inline-block mr-6">
+            border-dashed
+          </Text>
+          <Text appearance="default" className="border border-dotted bg-secondary-lightest p-8 d-inline-block mr-6">
+            border-dotted
+          </Text>
+          <Text appearance="default" className="border border-inset bg-secondary-lightest p-8 d-inline-block mr-6">
+            border-inset
+          </Text>
+          <Text appearance="default" className="border border-outset bg-secondary-lightest p-8 d-inline-block">
+            border-outset
+          </Text>
+        </div>
+        <div className="mb-5 bg-secondary-lightest p-3 rounded-10 font-size-xs">
+          <pre className="m-0">
+            <code className="d-block">
+              {
+                '<Text appearance="default" className="border border-solid bg-secondary-lightest p-8 d-inline-block mr-6">border-solid</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border border-dashed bg-secondary-lightest p-8 d-inline-block mr-6">border-dashed</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border border-dotted bg-secondary-lightest p-8 d-inline-block mr-6">border-dotted</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border border-inset bg-secondary-lightest p-8 d-inline-block mr-6">border-inset</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border border-outset bg-secondary-lightest p-8 d-inline-block">border-outset</Text>'
+              }
+            </code>
+          </pre>
```

Comment:
Instead of using multiple <code> elements we can simply write all the text inside single <code>